Updated ILO embeddings

dwnoble opened the pull request:
Updated iLO embedings to reflect stat var grouping changes from #4430. Test in explore page with URL params `dc=undata_dev` or `dc=undata_ilo`.

Example: http://localhost:8080/explore#q=unemployment+in+the+usa&client=ui_query&dc=undata_dev

<img width="1397" alt="Screenshot 2024-07-03 at 4 13 38 AM" src="https://github.com/datacommonsorg/website/assets/13766/32228e74-720b-4cc0-8bed-04e3674032f7">

shifucun commented on tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:
```diff
@@ -1,86 +1,33 @@
-dc/topic/ILOEIP3WAPRT,"Number of Young Persons of Working Age Outside The Labour Force, as A Percentage of The Working-age Population Youth Inactivity Rate"
+dc/topic/ILOEAR4MMNNB,"Monthly Minimum Wage"
+dc/topic/ILOEARXTLPRT,"Low Pay Rate"
+dc/topic/ILOEESXTMPRT,"Share of Temporary Employees"
```
"percentage of.." if that's the intention. "share of" might not result in good match.

dwnoble replied:
Added to alternatives.

shifucun commented on tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:
```diff
@@ -1,86 +1,33 @@
-dc/topic/ILOEMPPIFLNB,Employment Outside The Formal Sector
-dc/topic/ILOEMPPIFLRT,Share of Employment Outside The Formal Sector
+dc/topic/ILOEMPPIFLNB,"Employment Outside The Formal Sector"
+dc/topic/ILOEMPPIFLRT,"Share of Employment Outside The Formal Sector"
```
same here

dwnoble replied:
Added to alternatives.

shifucun commented on tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:
```diff
@@ -1,86 +1,33 @@
-dc/topic/ILOEMPNIFLNB,Informal Employment
-dc/topic/ILOEMPNIFLRT,Informal Employment Rate
+dc/topic/ILOEMPNIFLRT,"Informal Employment Rate"
 dc/topic/ILOEMPNORMNB,"Employment, Normative Approach"
```
"Employment, Normative Approach" is a bit vague?

dwnoble replied:
Added to alternatives.

shifucun commented on tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:
```diff
@@ -1,86 +1,33 @@
+dc/topic/ILOGED2LFPRT,"Prime-age Labour Force Participation Rate of Couple Households With Children Under Age 6 ILO Modelled Estimates"
+dc/topic/ILOINJDAYSNB,"Days Lost Due To Cases of Occupational Injury With Temporary Incapacity for Work"
+dc/topic/ILOINJFATLNB,"Cases of Fatal Occupational Injury"
+dc/topic/ILOINJFATLRT,"Fatal Occupational Injuries Per 100'000 Workers"
```
100'000 -> 100000?

dwnoble replied:
Added to alternatives, and gave this feedback to UN

shifucun commented on tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:
```diff
@@ -1,86 +1,33 @@
+dc/topic/ILOINJNFTLRT,"Non-fatal Occupational Injuries Per 100'000 Workers"
+dc/topic/ILOLAC4HRLNB,"Mean Nominal Hourly Labour Cost Per Employee"
+dc/topic/ILOLAIINSPNB,"Number of Labour Inspectors"
+dc/topic/ILOLAP2LIDRT,"Labour Income Distribution  ILO Modelled Estimates"
```
extra space here

dwnoble replied:
Gave this feedback to UN

shifucun commented on tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:
```diff
@@ -1,86 +1,33 @@
+dc/topic/ILOLUUXLU2RT,"Combined Rate of Time-related Underemployment And Unemployment LU2"
+dc/topic/ILOLUUXLUXNB,"Jobs Gap"
+dc/topic/ILOLUUXLUXRT,"Jobs Gap Rate"
+dc/topic/ILOSTRDAYSNB,"Days Not Worked Due To Strikes And Lockouts"
```
Number of Days? Give the next one are very similar, this needs to be distinct enough.

dwnoble replied:
Added to alternatives

shifucun commented on tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:
```diff
@@ -1,86 +1,33 @@
+dc/topic/ILOINJDAYSNB,"Days Lost Due To Cases of Occupational Injury With Temporary Incapacity for Work"
+dc/topic/ILOINJFATLNB,"Cases of Fatal Occupational Injury"
+dc/topic/ILOINJFATLRT,"Fatal Occupational Injuries Per 100'000 Workers"
+dc/topic/ILOINJNFTLNB,"Cases of Non-fatal Occupational Injury"
```
for this and some other similar ones, if they are count, maybe add "number of xxx"?

dwnoble replied:
Added to alternatives

shifucun commented:
the nl test error indicates updating golden files.

dwnoble commented:
Thanks @shifucun ! Updated the golden files, and i added all of your suggestions to an "alternatives.csv" file in the ilo nl descriptions directory. I also passed along all of these suggestions to the UN, and once they add them to the stat var descriptions and variable groupings, i'll re-generate these base NL descriptions.

shifucun reviewed (approved):
thanks for the update!
